bmeg_app/views/compare_dresp_view.py

```diff
@@ -21,12 +21,36 @@
 styles=ly.styles
 
 #######
-# Page  
-####### 
-print('loading app layout')   
-tab_layout = html.Div(children=[
+# Page
+#######
+print('loading app layout')
+NAME = "Compare Drug Responses"
+LAYOUT = html.Div(children=[
     dbc.Row(
         [
+            dbc.Col(
+                html.Div([
+                    dbc.Button('Info', id='open1',color='primary',style={'font-size':styles['t']['size_font']}),
+                    dbc.Modal(
+                        [
+                            dbc.ModalHeader('Identify Drugs Candidates with Similar Cell Reponses'),
+                            dbc.ModalBody('Interrogate cell line drug screening trials. For example, select a FDA drug that is widely known to treat a particular disease (Paclitaxel for breast cancer treatment) and identify other drugs that show a similar impact on cell lines.'),
+                            dbc.ModalBody( 'What’s going on behind the scenes?'),
+                            dbc.ModalBody( '•	Data is queried from BMEG, filtered for relevant cell lines (breast tissue derived cell lines kept if breast cancer is selected), and analyzed in the viewer.'),
+                            dbc.ModalBody( 'Features'),
+                            dbc.ModalBody( '• Boxed in blue is a summary of the selected drug to provide insight on the molecular and/or biological realm of the selected drug.'),
+                            dbc.ModalBody( '• Violin plots comparative overview: Identify alternative drugs by examining drug responses for similar distributions.'),
+                            dbc.ModalBody( '• Examine the drug characteristics table for taxonomic reasons for similar and different responses from drugs.'),
+                            dbc.ModalBody( '• Dive deeper to see the characteristics of the samples that generated the violin plots of particular drugs.'),
+                            dbc.ModalFooter(dbc.Button('Close',id='close1',className='ml-auto')),
+                        ],
+                        id='main_help1',
+                        size='med',
+                        centered=True,
+                    ),
+                ]),
+                width=1,
+            ),
```

Comment:
remove these lines. old code that has been moved elsewhere now

Reply:
Removed.